### PR TITLE
OCPVE-295: rteval

### PR DESCRIPTION
**Pull request by jeff-roche**
This implements rteval as a realtime kernel test and updates the latency thresholds to be a little more forgiving

### CODE REVIEW

File: test/extended/kernel/tools.go [hunk 1]
```diff
@@ -185,3 +188,54 @@ func getProcessorCount(oc *exutil.CLI) (int, error) {
...
+		Range                 int      `xml:"range"`
+		Mean                  float32  `xml:"mean"`
+		MeanAbsoluteDeviation float32  `xml:"mean_absolute_deviation"`
+		StandardDeviation     float32  `xml:"standard_deviation"`
+	} `xml:"Measurements>Profile>cyclictest>system>statistics"`
+}
+
+func runRteval(oc *exutil.CLI) error {
+	// Create a working directory
+	rtevalWorkDir := "/tmp"
```
Comment (eggfoobar) [inline test/extended/kernel/tools.go:210]: Small nit, using `os.TempDir()` would be better here to avoid someone else doing things in that dir.

Comment (jeff-roche) [reply test/extended/kernel/tools.go:210]: I'll update this comment, os.TempDir() would create a temp dir on the machine running the tests, not the machine the tests are being run against

Comment (eggfoobar) [reply test/extended/kernel/tools.go:211]: Thanks! I missed that part, lgtm to me otherwise.

**Comment by jeff-roche** [comment]
@bparees are you able to approve this one?

**Pull request merged**